packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -247,6 +222,13 @@ function Element({
           isCurrentResourceUploading(resourceId)) && (
           <LoadingBar loadingMessage={__('Uploading media…', 'web-stories')} />
         )}
+        <InsertionMenu
```

Comment:
💭  When we had to set up buttons that appeared over an image that had it's own functionality in various iterations of the dashboard what we ended up doing was just visually overlaying them and letting the html hierarchy fall like this: 
- container that is also a button that can get clicked on and used 
  - image shown for that container 
- another button 
- another button

```
<myLayeredDesign>
  <firstInteractiveElementContainer>
    <whateverINeedNestedToGetCSSWorking /> 
  </firstInteractiveElementContainer> 
  <WrapperToEaseOverlayFunctionality>
    <visuallyOnTopOfFirstInteractiveElement /> 
    <visuallyOnTopOfFirstInteractiveElement />
  </WrapperToEaseOverlayFunctionality>
</myLayeredDesign>
```

This makes the tab order way easier to preserve and allows screenreaders and keyboard users to have easier HTML outline from the get go.